server/tasks/process_file.py

```diff
@@ -38,3 +42,58 @@ def process_file_streaming(self, serialized_agent: bytes):
 
     # Run the agent
     asyncio.run(agent.arun())
+
+
+async def async_process_streaming_output(input_job_id: str, result_handler: str, batch_size: int):
+    logger.info(f"Polling for results {input_job_id=}")
+
+    try:
+        result_handler = ResultHandler.__dict__[result_handler]
```

Comment:
nit: you should be able to just do `ResultHandler[result_handler]` here. Just tested in ipython, and it worked.

Reply:
No, it doesnt when its a function in the enum. Try it